recipes/gnumpy/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "gnumpy" %}
+{% set version = "0.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "070b78ee648c42e3dcd2f56a4308cc26bcaa1de30a4f7c078eb5828050a4ec4f" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
```

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

and you can delete the ``file_ext`` variable

---

I'm starting to wonder what the point is of `conda skeleton pypi`